modules/media/php/NDB_Menu_Filter_media.class.inc

```diff
@@ -91,13 +91,12 @@ class NDB_Menu_Filter_Media extends NDB_Menu_Filter
             // allow only to view own site data
             $siteList =array();
```

Comment:
Could you maybe refactor this, so that `$siteList  = array();` is defined on top? E.g: 
```php
$siteList = array();
if ($user->hasPermission('access_all_profiles')) {
  $siteList  = Utility::getSiteList(false);
  // ...all sites loop...
} else {
  $site_arr = $user->getData('CenterIDs');
  // ...your code..
}